### ISSUE TITLE
cards are not registered/the decktracker ui is static and don't change

### ISSUE THREAD

**Issue by reslerr** [HearthSim/Hearthstone-Deck-Tracker#447]
Hi, I have downloaded the progam and I have a pretty big issue.

basically when I start hearthstone the ui of the program will appear(on top left the word hand, deck and the percentages, same on the right), so no problems here.

if I build a deck in the decktracker also the deck will show up in the ui on my right.

the problem is however that the program doesn't change when I start a game, my deck will stay static, and cards played by my opponent will not be registered(the left of hearthston will so stay empty)

basically the program doesn't react/notice that a game has started and will always stay static.

If anyone could help me I'd be grateful.
